e2e/cypress/integration/channel/header_spec.js

```diff
@@ -29,6 +30,7 @@ describe('Header', () => {
         cy.updateChannelHeader('>' + ' newheader'.repeat(20));
 
         // * Check if channel header description has ellipsis
+        cy.get('#channelHeaderDescription').should('have.text', ' newheader'.repeat(20));
```

Comment:
Added extra test so that the next verification to width is actually based on the new header and not on previous change.